src/components/network/NetworkGraphToolbar.vue

```diff
@@ -68,6 +68,29 @@
         </span>
 
         <span>
+          <v-dialog max-width="450" v-model="state.dialog">
+            <template #activator="{ on, attrs }">
+              <v-btn
+                :disabled="state.network.isEmpty()"
+                icon
+                small
+                title="Delete network"
+                v-bind="attrs"
+                v-on="on"
+              >
+                <v-icon small v-text="'mdi-trash-can-outline'" />
+              </v-btn>
+            </template>
+            <v-card class="about-dialog">
+              <v-card-title v-text="'Are you sure to delete network?'" />
+              <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn @click="state.dialog = false" text v-text="'close'" />
```

Comment:
I would rather opt for "No" as the opposite of "Yes". The dialog is also closed, when a user clicks on "yes", but the difference is the (non-)execution of the delete operation. I see some sort of difference compared to the the case when deleting a node, since the node context menu has already that structure in which a nested element uses a "BACK" button, together with the  arrow tip symbol.
```suggestion
                <v-btn @click="state.dialog = false" text v-text="'no'" />
```